AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py

```diff
@@ -0,0 +1,160 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+class Tests:
+    creation_undo = (
+        "UNDO Entity creation success",
+        "UNDO Entity creation failed")
+    creation_redo = (
+        "REDO Entity creation success",
+        "REDO Entity creation failed")
+    hdri_skybox_entity_creation = (
+        "HDRi Skybox successfully created",
+        "HDRi Skybox failed to be created")
+    hdri_skybox_entity_component_added = (
+        "Entity has an HDRi Skybox component",
+        "Entity failed to find HDRi Skybox component")
+    enter_game_mode = (
+        "Entered game mode",
+        "Failed to enter game mode")
+    exit_game_mode = (
+        "Exited game mode",
+        "Couldn't exit game mode")
+    is_visible = (
+        "Entity is visible",
+        "Entity was not visible")
+    is_hidden = (
+        "Entity is hidden",
+        "Entity was not hidden")
+    entity_deleted = (
+        "Entity deleted",
+        "Entity was not deleted")
+    deletion_undo = (
+        "UNDO deletion success",
+        "UNDO deletion failed")
+    deletion_redo = (
+        "REDO deletion success",
+        "REDO deletion failed")
+
+
+def AtomEditorComponents_HDRiSkybox_AddedToEntity():
+    """
+    Summary:
+    Tests the HDRi Skybox component can be added to an entity and has the expected functionality.
+
+    Test setup:
+    - Wait for Editor idle loop.
+    - Open the "Base" level.
+
+    Expected Behavior:
+    The component can be added, used in game mode, hidden/shown, deleted, and has accurate required components.
+    Creation and deletion undo/redo should also work.
+
+    Test Steps:
```

Comment:
I reviewed the linked test case ID on our test case repository and I noticed that there is a step to "Add a Cubemap Texture to the HDRi Skybox component." - shouldn't we add that step to these test steps as well?
Also I noticed the test case uses `\lightingpresets\greenwich_park_02_4k_iblskyboxcm.exr.streamingimage` - we actually can't use this any more. Instead use these alternatives (im fixing similar tests right now that also use t hat image - it's far too large, we can use the smaller version for our tests)

Reply:
I think that is old, since it is P0 we only do the required components to get it working. removing this steps from the TestRail case

Reply:
Per our discussion I still think this is worth having

Reply:
also wanted to mention, for the asset issue you can just cut a ticket for that to do in a different PR since it's out of scope
but we should def still add the missing test case step